howManyHours.js

```diff
@@ -0,0 +1,6 @@
+const howManyHours = (hours) => {
```

Comment:
Instructions unclear on how to execute this file. Please add instructions to `README`, or add an `npm` `package.json` with clear scripts.